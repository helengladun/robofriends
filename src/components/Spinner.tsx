import React from 'react'

export const Spinner = () => (
  <div className="full-screen flex justify-center items-center">
    <div className="pacman">
      <div className="pacman__mouth"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
    </div>
  </div>
);