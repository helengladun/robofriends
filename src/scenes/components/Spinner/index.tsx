import React from 'react'

const Spinner = () => (
  <div className="full-screen flex justify-center items-center">
    <div className="pacman">
      <div className="pacman__mouth"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
      <div className="pacman__food"></div>
    </div>
  </div>
);

export default React.memo(Spinner)

Spinner.whyDidYouRender = true;