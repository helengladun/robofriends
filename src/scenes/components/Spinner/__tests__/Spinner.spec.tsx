import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../index';

describe('Spinner component', () => {
  it('render Spinner component correctly', () => {
    expect(shallow(<Spinner />)).toMatchSnapshot();
  });
});
