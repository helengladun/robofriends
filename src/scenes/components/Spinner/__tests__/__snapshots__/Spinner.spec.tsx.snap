// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component render Spinner component correctly 1`] = `
<div
  className="full-screen flex justify-center items-center"
>
  <div
    className="pacman"
  >
    <div
      className="pacman__mouth"
    />
    <div
      className="pacman__food"
    />
    <div
      className="pacman__food"
    />
    <div
      className="pacman__food"
    />
    <div
      className="pacman__food"
    />
  </div>
</div>
`;
